test/commands/create.js

```diff
@@ -15,7 +16,7 @@ describe('apify create', () => {
         sinon.spy(console, 'log');
     });
 
-    ['superlongdescriptiontotesthtelengthofthename', 'sh', 'bad_escaped'].forEach((badActorName) => {
+    [_.times(151, () => 'a').join(''), 'sh', 'bad_escaped'].forEach((badActorName) => {
```

Comment:
how come people of apify are such in love with underscore :D 
```suggestion
    ['a'.repeat(151), 'sh', 'bad_escaped'].forEach((badActorName) => {
```